mne/report.py

```diff
@@ -154,6 +158,10 @@ def _iterate_trans_views(function, **kwargs):
     for view, ax in zip(views, axes):
         mlab.view(view[0], view[1])
         if fig.scene is not None:
+            try:
+                mlab.process_ui_events()
```

Comment:
not sure if this helps or if it's the same. I have this in one of my scripts:

```py
from pyface.api import GUI
GUI().process_events()
```

copied from some github issue ...